node_modules/bootstrap-multiselect/dist/js/bootstrap-multiselect.js

```diff
@@ -0,0 +1,1416 @@
+/**
```

Comment:
This file should not be here (if you have made edits here, they will get overwritten when other people run ```npm install```.

If you have edits, let's sit down to monkey patch in a better way!